.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 package-lock.json
-data/config.json
-data/rei.sqlite3
\ No newline at end of file
+data

```

Comment:
Why add this?

Reply:
Because why have `config.json` and `rei.sqlite3` separately when you can ignore `data` dir

Reply:
Because we might (and will!) add data in there that will be required to even get the bot running. It just currently contains data that we don't want to show.

Reply:
Ah so if that's the case then I'm really sorry for that